docs/cloudera/_index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Cloudera"
+bookIconImage: ../img/cloudera-logo.png
+bookFlatSection: true
+weight: 410
```

Comment:
Sure, will change it. Maybe worth adding a comment in a readme so that others will know about this requirement too? (unless this becomes obsolete with the new docs page design anyways)